src/Agent/NewRelic/Agent/Extensions/Providers/Wrapper/Bedrock/InvokeModelAsyncWrapper.cs

```diff
@@ -166,8 +177,9 @@ private void HandleError(ISegment segment, InvokeModelRequest invokeModelRequest
             }
 
             IResponsePayload responsePayload = null;
-            InvokeModelResponse invokeModelResponse = null;
+            dynamic invokeModelResponse = null;
 
+            // TODO: Figure out what the correct completion event is when there's an error. This call currently throws an exception because invokeModelResponse is null.
```

Comment:
There's already a ticket #2295 for the error response. I just wanted to make it clear where things fall apart, since the exception gets eaten by the wrapper.